packages/components/src/root-container/prop-builders.ts

```diff
@@ -0,0 +1,57 @@
+import { DEFAULT_DOM_OPTIONS, DOM_ATTRIBUTES, DOM_CSS_CLASSES } from './consts';
+import { DomOptions, PropBuilders } from './types';
+import { buildAttributes, buildCss, createBuilder } from './utils';
+
+export const PROP_BUILDERS: PropBuilders = {
+  accessKey: createBuilder(
+    DEFAULT_DOM_OPTIONS.accessKey,
+    (
+      { shortcutKey }: DomOptions['accessKey'],
+    ) => buildAttributes(!!shortcutKey, { [DOM_ATTRIBUTES.accessKey]: shortcutKey }),
+  ),
+  active: createBuilder(
+    DEFAULT_DOM_OPTIONS.active,
+    (
+      { activeStateEnabled }: DomOptions['active'],
+    ) => buildCss(activeStateEnabled, DOM_CSS_CLASSES.active),
+  ),
+  attributes: createBuilder(
+    DEFAULT_DOM_OPTIONS.attributes,
+    (
+      { attributes }: DomOptions['attributes'],
+    ) => buildAttributes(true, attributes),
+  ),
+  disabled: createBuilder(
+    DEFAULT_DOM_OPTIONS.disabled,
+    (
+      { disabled }: DomOptions['disabled'],
+    ) => buildAttributes(disabled, {
+      [DOM_ATTRIBUTES.disabled]: ' ',
+      [DOM_ATTRIBUTES.tabIndex]: undefined,
```

Comment:
Question: What is happen if we set tabIndex and disabled simultaniosly? Should we add some additional logic here?